File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> set = Arrays.stream(nums).boxed().collect(Collectors.toSet());

        int longest = 0;
        for(Integer num: set) {

            
            if(!set.contains(num - 1)) {// It's a sequence
                int length = 1;
                while(set.contains(num + length)) {
                    length++;
                }

                longest = Math.max(longest, length);
            }

        }

        return longest;

    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        
        counter = 0
        max_consecutive_1s = 0
        for i in range(len(nums)):
            if nums[i] == 1:
                counter += 1
            else:
                max_consecutive_1s = max(max_consecutive_1s, counter)
                counter = 0

        max_consecutive_1s = max(max_consecutive_1s, counter)
        return max_consecutive_1s
